pkg/vfs/iter.go

```diff
@@ -0,0 +1,100 @@
+package vfs
+
+import (
+	"errors"
+
+	"github.com/cozy/cozy-stack/pkg/consts"
+	"github.com/cozy/cozy-stack/pkg/couchdb"
+	"github.com/cozy/cozy-stack/pkg/couchdb/mango"
+)
+
+// ErrIteratorDone is returned by the Next() merthod of the iterator when
+// the iterator is actually done.
+var ErrIteratorDone = errors.New("No more element in the iterator")
+
+// IteratorDefaultFetchSize is the default number of elements fetched from
+// couchdb on each iteration.
+const IteratorDefaultFetchSize = 100
```

Comment:
This is the number of elements fetched from the database on each roundtrip of the iterator. The iterator is not only used by the pagination of the `web/files` package, it is sometime used to iterate over all the elements of the directory.

In the paginated case, where we potentially stop iterating before `ErrIteratorDone`, the `ByFetch` is less than this default and the pagination is only one roundtrip with `couchdb`. See [here](https://github.com/jinroh/cozy-stack/blob/040664d4d210d509751fabc9d058bdb0b122f86c/web/files/paginated.go#L46-L51).